tests/xtimer_mutex_lock_timeout/main.c

```diff
@@ -42,9 +49,38 @@ static const shell_command_t shell_commands[] = {
       cmd_test_xtimer_mutex_lock_timeout_long_unlocked, },
     { "mutex_timeout_long_locked", "locked mutex with long timeout",
       cmd_test_xtimer_mutex_lock_timeout_long_locked, },
+    { "mutex_timeout_long_locked_low",
+      "locked mutex from lower prio thread function with long timeout",
+      cmd_test_xtimer_mutex_lock_timeout_low_prio_thread, },
     { NULL, NULL, NULL }
 };
 
+/**
+ * @brief   thread function and stack for
+ *          cmd_test_xtimer_mutex_lock_timeout_low_prio_thread
+ */
+
```

Comment:
please remove empty line between doc and variable declaration

Reply:
Ok.